Codigo/CharacterPersistence.bas

```diff
@@ -237,21 +237,15 @@ Public Function LoadCharacterFromDB(ByVal userIndex As Integer) As Boolean
 
             'User pets
             Set RS = Query("SELECT number, pet_id FROM pet WHERE user_id = ?;", .ID)
-
+            .NroMascotas = 0
```

Comment:
We should not initialize fields in here as this is a sub to load the info from the DB

I would move this `.NroMascotas = 0` to `ConnectUser_Prepare()`